File: packages/volume/ios/Plugin/Classes/Results/IsWatchingResult.swift
import Foundation
import Capacitor

@objc public class IsWatchingResult: NSObject, Result {
    let watching: Bool

    init(watching: Bool) {
        self.watching = watching
    }

    @objc public func toJSObject() -> AnyObject {
        var result = JSObject()
        result["watching"] = watching
        return result as AnyObject
    }
}

File: packages/volume/android/src/main/java/io/capawesome/capacitorjs/plugins/volume/classes/results/IsWatchingResult.java
package io.capawesome.capacitorjs.plugins.volume.classes.results;

import androidx.annotation.NonNull;
import com.getcapacitor.JSObject;
import io.capawesome.capacitorjs.plugins.volume.interfaces.Result;

public class IsWatchingResult implements Result {

    private final boolean watching;

    public IsWatchingResult(boolean watching) {
        this.watching = watching;
    }

    @Override
    @NonNull
    public JSObject toJSObject() {
        JSObject result = new JSObject();
        result.put("watching", watching);
        return result;
    }
}
